desktop/components/commercial_filter/filters/location/location_filter_view.coffee

```diff
@@ -54,7 +54,7 @@ module.exports = class LocationFilterView extends Backbone.View
       return if _.contains(@params.get('partner_cities'), selectedCity)
       partner_cities = @params.get('partner_cities').concat selectedCity
       @additionalLocations.push selectedCity
-      @params.set { partner_cities: partner_cities, aggregation_partner_cities: @additionalLocations }
+      @params.set { partner_cities: partner_cities }
```

Comment:
You can probably leave this as-is. Some of the 'extra' aggregations, like mediums and additional cities you add to the sidebar, only get computed in Gravity/ES if you also include it in `aggregations: [PARTNER_CITY]` (for example).

So, since we removed it from the aggregations, we can probably just leave some of the faffing with the 'extra' aggregations- they won't be evaluated.  #minor

Reply:
Oh, good catch. This is a leftover from my first attempt to remove aggregations.